data-validation/use-cases.md

```diff
@@ -2,15 +2,148 @@
+- Most importantly, they don't have much recourse. Fixing the problem involves manually editing linked data, possibly requiring information lost when BarChat wrote malformed data. They are savvy, but they aren't experts in RDF.
+
+__An application *migrates* data, and creates breaking changes for other applications.__
+- Alice uses FooHealth to manage all of her personal health data, which is stored in her Solid pod.
+- FooHealth has a major update. For that update to work, it needs to cycle through all of her existing data and make structural changes.
+- FooHealth was already granted write access to Alice's health data because it is her primary tool for managing the data inside.
+- Alice gets a prompt from FooHealth to migrate her data. She approves it because she doesn't realize that it could affect other important applications that rely on that data to be in a consistent format.
+- The data is migrated, and Alice immediately realizes that two other important applications, TrueHealth and YouHealth, no longer work.
+
+__An application prompts a user to make decisions about shape constraints__
+- Bob has a brand new data pod, and has recently decided to use ExampleApp to manage the data he's collecting as part of research and development work that he's performing.
+- ExampleApp works with a wide variety of new shapes that Bob hasn't stored before, and writes data to the pod that conforms to different combinations of these shapes depending on the workflows Bob uses.
+- Before ExampleApp starts writing data, it prompts Bob and asks if he would like to enforce shape constraints on this data in his pod.
+- Based on the shapes it will write data for, and the places it will store them, ExampleApp prompts Bob with a bunch of toggles to determine which ones he does and doesn't want to enforce.
+- Bob has no idea what shapes are. He get intimidated and abandons ExampleApp and Solid.
 
 ## Use Cases
 
-The following use cases must be considered when designing a system for shape-based data validation in Solid.
+### Enabling schema validation
+
+These use cases are focused on *setting* or *enabling* schema validation for resources or containers.
+
+#### Enabling shape validation on a resource
+
+- Bob uses FooChat to communicate with his co-workers.
+- Bob creates a chatroom where he can exchange messages with Alice and Mary.
+- FooChat writes data conforming to a chat shape at `<https://bob.example/chats/maryalice.ttl>`, and turns on shape validation for `<https://bob.example/chats/maryalice.ttl>` to ensure that data in this resource conforms to the Chat shape.
+
+#### Enabling shape validation on a container
+
+- Bob uses FooChat to communicate with his co-workers.
+- Bob has a chatroom where he can exchange messages with Alice and Mary.
+- When Bob created the chatroom, FooChat setup a container to store messages.
+- The container organizes messages into sub-containers by month and day
+  - `<https://bob.example/chats/maryalice/2019/8/1.ttl>`
```

Comment:
We discussed this live in the panel meeting but recording here before resolving. Data discovery and footprints should be used to find things / identify where to write things and that's outside of the scope of validation so not trying to solve it here (though it will be solved elsewhere)